macros/form.njk

```diff
@@ -78,10 +78,12 @@
   {{ input(label, name, form, hint, 'email') }}
 {% endmacro %}
 
-{% macro textArea(label, name, form, inputClass='form-control-3-4', rows=5, labelClass='') %}
+{% macro textArea(label, name, form, inputClass='form-control-3-4', rows=5, labelClass='', bold = true) %}
```

Comment:
did you check if there's any places that shouldn't be bold? in the legal frontend as well?
The usual approach would be for non breaking changes, i.e. defaulting to false here

Reply:
That's our bad, let me fix that.